packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1261,7 +1275,7 @@ export class DeltaManager
         this.scheduleSequenceNumberUpdate(message, result.immediateNoOp === true);
 
         const endTime = Date.now();
-        this.emit("processTime", endTime - startTime);
```

Comment:
made a simlilar comment below. the other options is to keep emitting the old event

Reply:
There is only one usage sample I found (in this repo), so I'd rather kill it right away. :)
But yes, if we want to be strict with backward compat, we would need to go through depreciation path